python/testSrc/com/jetbrains/python/testing/PyTestFixtureAndParametrizedTest.kt

```diff
@@ -45,6 +45,39 @@ class PyTestFixtureAndParametrizedTest : PyTestCase() {
     myFixture.checkResultByFile("after_test_test.txt")
   }
 
+  fun testUsefixturesCompletionSuggestsFixture() {
```

Comment:
please use `TestFor` annotation
```suggestion
  @TestFor(issues = ["PY-54771"])
  fun testUsefixturesCompletionSuggestsFixture() {
```